contracts/6/forwarder/ERC20FeeProxy.sol

```diff
@@ -72,8 +82,12 @@ contract ERC20FeeProxy is ERC20ForwardRequestTypes,Ownable{
      * - etc
      */
      /// @param _transferHandlerGas : max amount of gas the function _transferHandler is expected to use
-    function setTransferHandlerGas(uint256 _transferHandlerGas) external onlyOwner{
-        transferHandlerGas = _transferHandlerGas;
+    function setTransferHandlerGas(address token, uint256 _transferHandlerGas) external onlyOwner{
+        transferHandlerGas[token] = _transferHandlerGas;
```

Comment:
Remove this file. as its renamed to ERC20Forwarder

Reply:
ok. done